internal/server/server_test.go

```diff
@@ -0,0 +1,121 @@
+package server_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/dicedb/dice/tests"
+	"net"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/dicedb/dice/config"
+)
+
+var options = tests.TestServerOptions{
```

Comment:
Let's call this testServerOptions instead. We should avoid overly generic names so that the namespace doesn't get polluted.

Reply:
Done